service.subtitles.legendasdivx/service.py

```diff
@@ -37,8 +37,12 @@
 
 __cwd__        = xbmc.translatePath(__addon__.getAddonInfo('path')).decode("utf-8")
 __profile__    = xbmc.translatePath(__addon__.getAddonInfo('profile')).decode("utf-8")
-__resource__   = xbmc.translatePath(pjoin(__cwd__, 'resources', 'lib' ) ).decode("utf-8")
-__temp__       = xbmc.translatePath(pjoin(__profile__, 'temp'))
+__resource__   = xbmc.translatePath(os.path.join(__cwd__, 'resources', 'lib' )).decode("utf-8")
```

Comment:
Ok.
Been like this since the service changed from Amet's plugin.
I'll do that on my other addons as well.
Thanks.